geode-core/src/main/java/org/apache/geode/distributed/ConfigurationProperties.java

```diff
@@ -1929,7 +1929,8 @@
    * The static String definition of the <i>"redis-bind-address"</i> property <a
    * name="redis-bind-address"/a>
    * </p>
-   * <U>Description</U>: Specifies the bind address used by Redis API for Geode
+   * <U>Description</U>: Specifies the bind address used by Redis API for Geode. If "" specified,
+   * the default is determined from the /etc/hosts file.
```

Comment:
I feel like how the default will be determined is too specific. I doubt we explicitly read the /etc/hosts file. It is probably something more like "the default will be requested from the operating system". It might be worth looking at our implementation and seeing what it actually does when the value is "".

Reply:
We use LocalHostUtil.getLocalHost() I like your wording.